analyzer-runtime/src/main/java/cloud/erda/analyzer/runtime/expression/functions/aggregators/Accumulator.java

```diff
@@ -39,7 +39,7 @@
 
     void setKey(String key);
 
-    void setAttribute(String key, String value);
+    void setAttribute(String key, Object value);
```

Comment:
why change attri value to Object? in Erda/streaming ,the assumption for the meric tags is map[string]string